dev/CommonStyles/APITests/CommonStylesTests.cs

```diff
@@ -127,7 +127,7 @@ public void VerifyVisualTreeExampleForLightTheme()
                 theme: Theme.Light);
         }
 
-        [TestMethod]
+        //[TestMethod] TODO: Re-enable once issue #645 is fixed.
```

Comment:
It's possible that

`return !(v.Equals(value) || (string.IsNullOrEmpty(value ) && value.StartsWith("Exception")));`

should be

`return !(v.Equals(value) || (!string.IsNullOrEmpty(value ) && value.StartsWith("Exception")));`